web/modules/custom/weather_block/weather_block.routing.yml

```diff
@@ -0,0 +1,7 @@
+weather_block.input_weather:
+  path: '/weather/select'
```

Comment:
From where user can access that?

Reply:
From user page, im added a block that shows a text with this url